pkg/stringHelpers/truncate.go

```
@@ -2,27 +2,35 @@
 	"fmt"
-	"strconv"
-
-	"github.com/sigurn/crc16"
+	"hash/crc32"
 )
 
-func TruncateWithHash(input string, maxLength int) string {
-	// If the input is already within the maxLength, no need to truncate
-	if len(input) <= maxLength {
-		return input
+// TruncateWithHash truncates a string to maxLength and appends a hash of the original string
```

golangci-lint (1.64.8)

[error] Comment should end in a period

(godot)